lang/main.json

```diff
@@ -223,7 +223,9 @@
         "grantedToUnknown": "Moderator rights granted to $t(notify.somebody)!",
         "muted": "You have started the conversation muted.",
         "mutedTitle": "You're muted!",
-        "raisedHand": "Would like to speak."
+        "raisedHand": "Would like to speak.",
+        "suboptimalExperienceTitle": "Browser Warning",
+        "suboptimalExperienceDescription": "Eer .. we are afraid your experience with __appName__ isn't going to be that great here. We are looking for ways to improve this but, until then, please try using one of the <a href='/static/recommendedBrowsers.html' target='_blank'>fully supported browsers</a>"
```

Comment:
I'm gonna be picky and say an ellipsis should consist of three dots. Also there's no period at the end of the sentence.